libraries/core-react/CHANGELOG.md

```diff
@@ -5,6 +5,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [0.16.1] - 2021-12-07
+
+### Changed
+
+- Bundle tree-shaked ramda instead of having it as dependency to avoid security warning for users ([#1768](https://github.com/equinor/design-system/issues/1768))
```

Comment:
maybe add; security warning for users by unused ramda functions? 😅

Reply:
just to clarify it was never really used, even though snyk flags the package as a whole...

Reply:
done😅